mantle/kola/cluster/cluster.go

```diff
@@ -103,13 +104,28 @@ func DropFile(machines []platform.Machine, localPath string) error {
 		if _, err := in.Seek(0, 0); err != nil {
 			return err
 		}
-		if err := platform.InstallFile(in, m, filepath.Base(localPath)); err != nil {
+		base := filepath.Base(localPath)
+		partial := base + ".partial"
+		if err := platform.InstallFile(in, m, partial); err != nil {
```

Comment:
the comment in the commit message about using a copy to a different file name and then replace seems appropriate. I think it would be useful to add somehwere in here. The "why" helps a lot. git commit messages are useful, but people don't always consider looking at them when they are changing things around.